clients/itwin/src/test/Client.test.ts

```diff
@@ -32,10 +32,11 @@ describe("Client", () => {
   });
 
   it("should apply prefix with config entry", async () => {
-    Config.App.set("imjs_url_prefix", "test-");
+    process.env.IMJS_URL_PREFIX = "test-";
     const requestContext = new ClientRequestContext();
     const url = await client.getUrl(requestContext);
     expect(url).to.equal("https://test-api.bentley.com/test-api");
-    Config.App.remove("imjs_url_prefix");
+    // eslint-disable-next-line dot-notation
+    delete process.env.IMJS_URL_PREFIX;
```

Comment:
Is this eslint disable comment needed?  This line looks like it is using dot notation as the rule suggests

Reply:
Good point. I think I originally had it in brackets and then just forgot to remove the comment after I changed it to dot notation. Thanks for pointing this out.